Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -297,6 +297,8 @@ var ScrollView = React.createClass({
     /**
      * A RefreshControl component, used to provide pull-to-refresh
      * functionality for the ScrollView.
+     *
+     * See [RefreshControl](/react-native/docs/refreshcontrol.html).
```

Comment:
This lives in a folder called react-native-github and we actively remove all internal documentation on components that exist in open source and link to the website instead.